index.js

```diff
@@ -181,3 +185,55 @@ AddonManager.addInstallListener({
     app.send('addon-install:download-failed', formatInstallData(install));
   }
 });
+
+function pingServer(title, addon) {
+  cookieManager.getCookiesFromHost('ideatown.dev', function(cookies) {
```

Comment:
Another note on this line: rather than try to get cookies every single time we ping the server, we could [listen for cookie changes](https://developer.mozilla.org/en-US/docs/Observer_Notifications#Cookies) and, if we don't currently have a session, save events in a queue that's persisted locally (maybe IndexedDB?) until the user logs back into the site. Then, we could send the queued events all at once.

This is a rather ambitious change, so probably best to file a followup issue for now :-)